### ISSUE TITLE
Remove Citadel mk.1

### ISSUE THREAD

**Issue by Kami-Interface** [Team-QUIET/QUIET-Community-Edition#114]

**Comment by PurpleMossClump** [comment]
Perhaps a look under the hood could do some good for this one? 
My main issue with the Citadel I is that it roams around when fighting. I'd prefer it behave more as an airborne battlestation than a weird giant gunship. 
(The Citadel II also likes to roam around when you give it an attack order.)

**Comment by Kami-Interface** [comment]
Created a discussion thread for this topic
https://github.com/LOUD-Community-Edition/LOUD-Community-Edition/discussions/131

Also Az said he'd try to fix the autonomous behavior of the different experimental units such as the Basilisk. Although this issue with the Citadels could have more to do with how air units like to follow units way too much

**Comment by PurpleMossClump** [comment]
Yeah, Gunships have a 'circling' behavior and I haven't found a way to stop them from doing it.

**Comment by Azraeel** [comment]
its in the Air Table, its called circling.
